app/scripts/views/sign_up.js

```diff
@@ -177,6 +181,14 @@ define(function (require, exports, module) {
         shouldFocusPassword: autofocusEl === 'password',
         showSyncSuggestion: this.isSyncSuggestionEnabled()
       };
+      if (this._webChannelSupportExists()) {
+        context.escapedSyncSuggestionUrl = encodeURI('/signup?service=sync&context=fx_desktop_v3&entrypoint=uitour');
```

Comment:
While I was looking at the URL generator [in connect_another-device.js](https://github.com/mozilla/fxa-content-server/blob/c57a9a7eaf1bbf73e1029bccb77600180c47f1c8/app/scripts/views/connect_another_device.js#L224:L246), I realized we also want to propagate the already existing `utm_` parameters. Like `entrypoint`, these parameters are used by our metrics to help bucket users. I wouldn't expect most users to have `utm_` parameters, we should propagate them anyways.